msteams-platform/toolkit/debug-overview.md

```diff
@@ -128,60 +124,120 @@ In `.fx/configs/tasks.json` under `"Validate & install prerequisites"` > `"args"
 
-In `.fx/configs/tasks.json`, set npmInstallArgs under `"Install npm packages"`.
+In `teamsapp.local.yml`, edit `args` of `cli/runNpmCommand` action.
+
+:::image type="content" source="../assets/images/teams-toolkit-v2/debug/customize-npm-install.png" alt-text="Install npm package":::
   
-   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/customize-npm-install.png" alt-text="Install npm package":::
+```yml
+# teamsapp.local.yml
+...
+  - uses: cli/runNpmCommand
+    with:
+      # edit the npm command args
+      args: install --no-audit
+...
+```
 
 </details>
 
 <details>
 <summary><b>Modify ports</b></summary>
 
 * Bot
-  1. Search for `"3978"` across your project and look for appearances in `tasks.json`, `ngrok.yml` and `index.js`.
+  1. Search for `"3978"` across your project and look for appearances in `tasks.json` and `index.js`.
   1. Replace it with your port.
+
      :::image type="content" source="../assets/images/teams-toolkit-v2/debug/modify-ports-bot.png" alt-text="Replace your port for bot":::
+
 * Tab
-  1. In `.fx/configs/tasks.json`, search for `"53000"`.
+
+  1. Search for `"53000"` across your project and look for appearances in `teamsapp.local.yml` and `tasks.json`.
   1. Replace it with your port.
+  
      :::image type="content" source="../assets/images/teams-toolkit-v2/debug/modify-ports-tab.png" alt-text="Replace your port for tab":::
 
 </details>
 
 <details>
 <summary><b>Use your own app package</b></summary>
 
-In `.fx/configs/tasks.json`, set `"appPackagePath"` under `"Build & upload Teams manifest"` to your app package's path.
-
-  :::image type="content" source="../assets/images/teams-toolkit-v2/debug/app-package-path.png" alt-text="use your own app package path":::
+Teams Toolkit by default creates a set of `teamsApp` actions to manage app package. You can update those in `teamsapp.local.yml` to use your own app package.
+
+:::image type="content" source="../assets/images/teams-toolkit-v2/debug/app-package-path.png" alt-text="use your own app package path":::
+
+```yml
+# teamsapp.local.yml
+...
+  - uses: teamsApp/create # Creates a Teams app
+    ...
+  - uses: teamsApp/validateManifest # Validate using manifest schema
+    ...
+  - uses: teamsApp/zipAppPackage # Build Teams app package with latest env value
+    ...
+  - uses: teamsApp/validateAppPackage # Validate app package using validation rules
+    ...
+  - uses: teamsApp/update # Apply the Teams app manifest to an existing Teams app in Teams Developer Portal.
+    ...
+...
+```
 
 </details>
 
 <details>
 <summary><b>Use your own tunnel</b></summary>
 
-1. In `.fx/configs/tasks.json` under `"Start Teams App Locally"`, you can update `"Start Local tunnel"`.
+1. In `.vscode/tasks.json` under `"Start Teams App Locally"`, you can update `"Start Local tunnel"`.
 
    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/start-local-tunnel.png" alt-text="Use your own tunnel":::
```

Comment:
Replace this screenshot with:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/37978464/3f8b8f75-0928-48da-907d-85a02fdc6c50)

Reply:
Updated.